server/rpc/filters/filters.go

```diff
@@ -120,6 +122,20 @@ func authUnaryServerInterceptor(env environment.Env) grpc.UnaryServerInterceptor
 	return contextReplacingUnaryServerInterceptor(ctxFn)
 }
 
+func roleAuthUnaryServerInterceptor(env environment.Env) grpc.UnaryServerInterceptor {
```

Comment:
no streaming filter is necessary? If we added a streaming RPC would it be unprotected?

Reply:
Good point, added a streaming filter too.